syntax = "proto3";

package auth;

option go_package = "user-service/proto";

service AuthService {
  rpc VerifyToken(VerifyTokenRequest) returns (VerifyTokenResponse) {}
}

message VerifyTokenRequest {
  string user_id = 1;
  string token = 2;
}

message VerifyTokenResponse {
  bool valid = 1;
  string error = 2;
} 
